aioxmpp/avatar/service.py

```diff
@@ -301,12 +301,19 @@ class AvatarClient(service.Service):
     Retrieves and caches avatar information of other clients. Emits
     notifications on metadata changes.
 
-    .. autosignal:: on_metadata_changed
+    .. note:: :class:`AvatarClient` only caches the metadata, not the
+              actual image data. This is the job of the caller.
+
+    .. signal:: on_metadata_changed(jid, metadata)
+
+        Fires when avatar metadata changes.
+
+        :param jid: The JID which the avatar belongs to.
+        :param metadata: The list of metadata descriptors.
```

Comment:
Add type information to the parameters, e.g. ``:type metadata: :class:`something` ``.